### DISCUSSION TITLE
Dispearing cursor when click mouse

### DISCUSSION THREAD

**Discussion by visuddhananda** [zim-desktop-wiki/zim-desktop-wiki#2064] [Q&A]
if click the blank space , cursor is dispeared. 
can not found where cursor is

**Comment by jaap-karssenberg** [comment]
Please explain a bit more

Op wo 29 jul. 2020 03:04 schreef ukebida <notifications@github.com>:

**Comment by visuddhananda** [comment]
If I type text and press 'enter' then mouse cursor(?or bar?) disappear. 
or If I type text and click the blank space then mouse cursor also disappear. 

It is very inconvenient. if I want to know the line number  I have to type the text.
When I use other text editor(MS word, Evernote, etc), I always can see the blinking bar like this 'I'


Windows 10 pro home 
version 1903
OS build 18362.959

**Comment by jaap-karssenberg** [comment]
Testing 0.73 here and do not see the issue.

What version of zim are you using?
Could it be related to keyboard input for non-english text or similar ?

-- Jaap

**Comment by visuddhananda** [comment]
Thank you. 
I am using 0.73.2. 
I have typed South Korean text.
But similar problem also occurs  when using English text. and that problem occurs even I do not type any word.
I don't know what the cause is. 
I don't kow keyboard input issue.

**Comment by notmikeb** [comment]
i have the same problem but i fixed this by changing it to another default font
edit /preference / use a custom font , i select another font and then it work wells
if i do not  chang it , the new line default height seems 0 pixels and the cursor seems gone

**Comment by eyolf** [comment]
Same problem here, under some circumstances: 
It happens when I click somewhere, 
It does NOT always happen, neither when I click nor always when I click the same place
the cursor always comes back when I move it with the arrow keys.
This happens whether or not I change the default font.

**Comment by Httqm** [comment]
My 0.02$, just in case that could help ONE person... :
I've just noticed that pressing the [F7] key toggles the cursor visibility.

As for the context, I'm using : 
* Ubuntu 20.04
* XFCE4
* Zim 0.73.5

**Comment by introt** [comment]
F7 is a Gtk shortcut for toggling cursor visibility, but is also the default accelerator for checking spelling - should probably switch that to eg. shift + F7 like Gedit does.
